src/controllers/MixcloudController.js

```diff
@@ -1,26 +1,48 @@
-controller = new BasicController({
-  supports: {
-    playpause: true,
-    previous: true,
-    next: true
+const isNewPlayer = () => !!document.querySelector('.mz-player-control');
+
+const config = [
+  {
+    test: () => !isNewPlayer(),
```

Comment:
In the case that we discussed, where there is a loading screen, this would pass even if it was a new player.  A better way to do it would be to have a positive case for both players, rather than depending on the other failing. 

Not a big deal for now, since as we said it doesn't matter too much for this particular case.